tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot

```diff
@@ -0,0 +1,28 @@
+*** Settings ***
+Library         SeleniumLibrary
+Resource        ../../../Resources/Page/OCPLogin/OCPLogin.resource
+Resource        ../../../Resources/OCP.resource
+Test Setup      Dashboard Test Setup
+Test Teardown   Dashboard Test Teardown
+
+*** Test Cases ***
+Verify if it is possible to install Pachyderm
+    [Tags]  ODS-1137
+    Navigate to OperatorHub
+    Get The Number of Operator Available    pachyderm
+    Install Operator    pachyderm
+    Navigate to Installed Operators
+    Search Installed Operator   pachyderm
+    Is Operator Installed   pachyderm
```

Comment:
Line number 11-16 could be replaced with **_Check And Install Operator in Openshift_**

Reply:
Tier2 @tarukumar